.travis.yml

```diff
@@ -25,6 +26,13 @@ before_install:
 - if $INSTALL_DEB_DEPENDECIES; then source /etc/fsl/fsl.sh; fi
 - if $INSTALL_DEB_DEPENDECIES; then source /etc/afni/afni.sh; fi
 - export FSLOUTPUTTYPE=NIFTI_GZ
+# Install vtk and fix numpy installation problem
```

Comment:
i don't think this is necessary since you should be able to install mayavi and traits also using conda (definitely for python 2), but i've also installed the pieces for python 3.

if conda hasn't released mayavi under python 3, one can install this version:

pip install https://github.com/enthought/mayavi/archive/a811639986fc1babecea68656f301c9a68d1ec07.zip
(it works with VTK 6, which is released for both python 2 and 3).

i think that will significantly simplify this file.